lib/finishing_moves/array.rb

```diff
@@ -14,7 +14,7 @@ def to_hash_values(starting_key = 0, &block)
 
   def to_indexed_hash(starting_key = 0)
     raise ArgumentError, "#{starting_key.inspect} is not an integer" unless starting_key.is_a? Integer
-    to_hash_values(starting_key) { |i| i + 1 }
```

Comment:
This was removed because this block is the default argument so the explicit nature is not needed.